gitdash/README.md

```diff
@@ -0,0 +1,58 @@
+This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
```

Comment:
You should think about replacing the main repo's `README.md` with this `gitdash/README.md` file :)

Reply:
We will do that!